src/fsi/fsiProject/fsi.fsproj

```diff
@@ -11,12 +11,14 @@
   <PropertyGroup Condition="'$(Configuration)' == 'Proto'">
     <TargetFramework>$(FSharpNetCoreProductTargetFramework)</TargetFramework>
     <PublishReadyToRun>true</PublishReadyToRun>
+    <PublishReadyToRunCrossgen2ExtraArgs>--embed-pgo-data</PublishReadyToRunCrossgen2ExtraArgs>
     <RuntimeIdentifier>$(NETCoreSdkRuntimeIdentifier)</RuntimeIdentifier>
   </PropertyGroup>
 
   <ItemGroup Condition="'$(Configuration)' == 'Proto'">
     <PublishReadyToRunExclude Include="FSharp.Build.dll" />
     <PublishReadyToRunExclude Include="FSharp.Core.dll" />
+    <PublishReadyToRunPgoFiles Include="$(MibcPath)" />
   </ItemGroup>
```

Comment:
Also here, we want it mostly in product. So needs to be enabled when packing product as well.